projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v3-endpoint-blogging-prompts.php

```diff
@@ -251,6 +253,36 @@ public function filter_sql( $clauses ) {
 		return $clauses;
 	}
 
+	/**
+	 * Prepares multiple blogging prompts output for response.
+	 *
+	 * @param array $items Prompt objects.
+	 * @return array A collection of prompt objects.
+	 */
+	public function prepare_items_for_response( $items ) {
+		$locale = get_locale();
+
+		if ( ! isset( $locale ) || empty( $locale ) ) {
```

Comment:
I think you could simplify this, `empty` already checks if `isset`:

```suggestion
		if ( empty( $locale ) ) {
```